UPGRADING.rst

```diff
@@ -10,6 +10,16 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#2025 Register indexer SA with Broad's SAM during deployment
+============================================================
+
+This PR introduces two new deployment-specific environment variables,
+``AZUL_TDR_SERVICE_URL`` and ``AZUL_SAM_SERVICE_URL``. Copy the settings for
+these variables from the example deployment to your personal deployment.
+
+Service accounts must be registered and authorized with SAM for integration
+tests to pass. See section 3.2.1 of the README for resgistration instructions.
```

Comment:
hyperlink (something that can be clicked) 

The link should be relative https://github.blog/2013-01-31-relative-links-in-markup-files/ and deep (point at the section heading).

Reply:
`UPGRADING.rst` is reStructuredText, not markup. The syntax in your link does not work. I can create a relative link to the README using a different syntax, but I see no way to create a _deep_ link, as rst does not understand markup document structure.

Reply:
Think this over one more time. The concept of relative links and deep links are HTTP concepts and have nothing to to with how the document is marked up. If you still can't figure it out, I'll do it during the review.